oauth2_http/java/com/google/auth/oauth2/DefaultCredentialsProvider.java

```diff
@@ -79,10 +79,17 @@
   /**
    * Returns the Application Default Credentials.
    *
-   * <p>Returns the Application Default Credentials which are credentials that identify and
-   * authorize the whole application. This is the built-in service account if running on Google
-   * Compute Engine or credentials specified by an environment variable or a file in a well-known
-   * location.</p>
+   * <p>Returns the Application Default Credentials which are used to identify and authorize the
+   * whole application. The following are searched (in order) to find the Application Default
+   * Credentials:
+   * <ol>
+   *   <li>Credentials file pointed to by the {@code GOOGLE_APPLICATION_CREDENTIALS} environment
+   *   variable</li>
+   *   <li>Google Cloud SDK ({@code gcloud}) credentials</li>
```

Comment:
This line item might be a bit misleading. What is actually searched is a well-known-file location which can be populated by tools like the Cloud SDK or Visual Studio. As of the last SDK, the main credential from "gcloud auth login" no longer populates this. The new command "gcloud auth application-default login does". So for this doc I'd suggest something like "Credentials provided by Cloud SDK gcloud auth application-default login command".